migrate_pim/apply_patch/apply_patch_ce.rst

```diff
@@ -0,0 +1,47 @@
+Guide to apply a patch for Community Edition
+=============================================
+
+Run the composer update command:
+
+.. code-block:: bash
+
+    composer --prefer-dist update
+
+Be aware that your composer.json won't be updated and some dependencies might be missing or from an outdated version.
+
+You have to check whether the latest composer.json is different from your own. In this case you should backup your current composer.json and download the newest one beforehand.
+
+Double check in the output of this command that the 2.3.10 version has been fetched, you can also check it by using the following command:
```

Comment:
2.3.10 is an example?

Reply:
yep... 
before we have this line : "In the following example, Akeneo PIM version 2.3.10 has just been released and we are using an Akeneo PIM version 2.3.9."
But I removed it on CE guide, maybe said something like "Double check in the output of this command that the latest version has been fetched, you can also check it by using the following command:"
What do you think ?